### PR TITLE
Add docstrings and unit tests for plugin

### PR DESCRIPTION
**Pull request by smarlowucf**

### CODE REVIEW

**Review by KeithMnemonic — APPROVED** [review]
LGTM

**Comment by smarlowucf** [comment]
Updated env vars to be generic and not NV specific.

**Review by brett060102 — APPROVED** [review]
looks good to me as well.

File: csp_billing_adapter_k8s/plugin.py [hunk 1]
```diff
@@ -35,14 +46,25 @@
 
 @csp_billing_adapter.hookimpl
 def setup_adapter(config: Config):
+    """
+    Authenticae to k8s cluster
```
Comment (rjschwei) [inline csp_billing_adapter_k8s/plugin.py:50]: s/Authenticae/Authenticate/

**Pull request merged**